package.json

```diff
@@ -475,7 +477,6 @@
     "trim-newlines": "3.0.1",
     "normalize-url": "4.5.1",
     "dicer": "0.3.0",
-    "xmldom": "github:xmldom/xmldom#0.7.0",
```

Comment:
This was added to resolve some security issue, so only removing this is not enough. 
I went through yarn.lock and removed and updated other stuff in this #13132, so we should merge that first, which cleans up this.